projects/packages/search/src/widgets/class-search-widget.php

```diff
@@ -164,7 +164,7 @@ public function widget_admin_setup() {
 	 * @since 5.8.0
 	 */
 	public function enqueue_frontend_scripts() {
-		if ( ! is_active_widget( false, false, $this->id_base, true ) || Options::is_instant_enabled() ) {
+		if ( Options::is_instant_enabled() ) {
```

Comment:
Should we remove this function from being hooked on L85 of this file?

Reply:
Yes, missed that! Done in fac6440b074d4f3c1a641ab9ebc14caf1c828bb7